packages/esm-patient-attachments-app/src/camera-media-uploader/paste-image-url.scss

```
@@ -0,0 +1,29 @@
+@use '@carbon/layout';
+@use '@openmrs/esm-styleguide/src/vars' as vars;
```

```suggestion
@use '@openmrs/esm-styleguide/src/vars' as *;
```

We can default to this as there's no risk of namespace collisions in this file.